### ISSUE TITLE
pkgs/development/node-packages/generate.sh fails because of quicktype

### ISSUE THREAD

**Issue by BeneSim** [NixOS/nixpkgs#211268]
### Steps To Reproduce
1. run `pkgs/development/node-packages/generate.sh`

Alternatively just run:
`"$(nix-build . -A nodePackages.node2nix)"/bin/node2nix -i <(echo '["quicktype"]')`
in a bash shell within nixpkgs.

### Build log
relevant part:
```
...
http 200 https://registry.npmjs.org/base64-js
info attempt registry request try #1 at 7:56:26 PM
http request GET https://registry.npmjs.org/unicode-trie
http 200 https://registry.npmjs.org/unicode-trie
info attempt registry request try #1 at 7:56:26 PM
http request GET https://registry.npmjs.org/pako
http 200 https://registry.npmjs.org/pako
info attempt registry request try #1 at 7:56:26 PM
http request GET https://registry.npmjs.org/tiny-inflate
http 200 https://registry.npmjs.org/tiny-inflate
fetching local directory: ../quicktype-core from quicktype/quicktype-graphql-input
Error: ENOENT: no such file or directory, open '/home/username/path/to/nixpkgs/pkgs/development/node-packages/quicktype/quicktype-core/package.json'
```

### Additional context
Seems to be related to the upstream package, haven't dug into it yet though.

### Notify maintainers
@SuperSandro2000 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.3, NixOS, 22.11 (Raccoon), 22.11.1629.2f9fd351ec3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"home-manager-22.05.tar.gz, nixos-22.11"`
 - channels(bisbarn): `"nixpkgs-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by BeneSim** [comment]
Alright, I was able to track it down to this change (18.0.6 -> 19.0.4)
https://github.com/quicktype/quicktype/compare/f347c71e607451804894fe26e04dcdc152ac47a0..20296629979c6c4d783f1b02075da80eb00ba166

In particular I guess this is problematic (at least with node2nix)
https://github.com/quicktype/quicktype/blob/20296629979c6c4d783f1b02075da80eb00ba166/packages/quicktype-graphql-input/package.json#L15

Easiest fix would be to pin `quicktype` to `18.0.6` until the problem is resolved. Opinions?

**Comment by BeneSim** [comment]
Issue has been fixed upstream :+1:
